protos/flyteidl/core/workflow.proto

```diff
@@ -205,3 +205,21 @@ message TaskNodeOverrides {
     // A customizable interface to convey resources requested for a task container. 
     Resources resources = 1;
 }
+
+// The workflow id is already used and the structure is different
+message WorkflowErrorExistsDifferentStructure {
+    string id = 1;
+}
+
+// The workflow id is already used with an identical sctructure
+message WorkflowErrorExistsIdenticalStructure {
```

Comment:
I wonder if we need this message since we return a different gRPC status code (AlreadyExists) for this scenario

Reply:
although perhaps it would be nice to keep this to include the existing workflow id 👍